content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -79,6 +79,30 @@ spec:
 EOF
 {{< /text >}}
 
+{{< tip >}}
```

Comment:
did you mean to have the <bash> section also inside the tip?

Reply:
Take a look at the preview here, it is not looking that great to have the tip like this: https://deploy-preview-12486--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/ I would rather say you just add an additional comment to the existing yaml at line 71 instead of duplicating the whole stuff.